test/blockstore-test.js

```diff
@@ -109,7 +109,7 @@ module.exports = (repo) => {
 
       it('massive read', (done) => {
         parallel(_.range(20 * 100).map((i) => (cb) => {
-          const j = i % 20
+          const j = i % blockCollection.length
```

Comment:
What value? This is iterating over the earlier put `blockCollection` doing parallel reads